client/components/common/tce-core/AddElement/SelectElement.vue

```diff
@@ -97,10 +107,6 @@ export default {
       this.type = null;
     }
   },
-  created() {
```

Comment:
We should have a way to only show question types if container was configured to only accept `ASSESSMENT` or `REFLECTION`.